docs/configuration.md

```diff
@@ -91,6 +92,17 @@ The availability zone names must be exposed as an environment variable `OPENSTAC
 
 The DNS servers must be exposed as an environment variable `OPENSTACK_DNS_NAMESERVERS`.
 
+## Machine flavor
+
+The flavors for control plane and worker node machines must be exposed as following environment variables 
+
```

Comment:
Please remove this bash script. We do not need to explain how to add these variables as OS environment variable explicitly.
Because variables are also given by not only OS environment but also the config file.
https://cluster-api.sigs.k8s.io/clusterctl/configuration.html#variables